config/locales/en/helpers.yml

```diff
@@ -382,6 +372,22 @@ en:
       steps_conviction_motoring_disqualification_end_date_form:
         motoring_disqualification_end_date: When did your disqualification end?
         approximate_motoring_disqualification_end_date: *approximate_date_legend
+      steps_conviction_conviction_subtype_form:
+        # default key
+        adult_financial: What type of order were you given?
```

Comment:
`adult_financial` and `prevention_reparation` should be replaced with:

```yml
default: What type of order were you given?
```

So the legend is shown if no specific key is defined.

Reply:
It was not catching the default, I probably need to change some code, I'll have a look now